### ISSUE TITLE
Errors if the DAGSTER_HOME path has spaces on a mac.

### ISSUE THREAD

**Issue by Jaikant** [dagster-io/dagster#3736]
## Summary
If the DAGSTER_HOME path, has spaces in the path, there are errors noticed while executing a pipeline on a mac

## Additional Info about Your Environment
I am executing the pipeline from within a python script, using the `execute_pipeline_iterator` 

Here are the logs - notice the space between "Application" and "Support" causing the `tail` to fail.

```
2021-02-25 16:14:17 - dagster - DEBUG - single_solid - df17c0a1-25d6-4a19-891a-5938d932ea60 - 15317 - process_file_alias - ENGINE_EVENT - Starting initialization of resources [io_manager].
DagsterEvent(event_type_value='ENGINE_EVENT', pipeline_name='single_solid', step_handle=StepHandle(solid_handle=SolidHandle(name='process_file_alias', parent=None)), solid_handle=None, step_kind_value=None, logging_tags={}, event_specific_data=EngineEventData(metadata_entries=[], error=None, marker_start='resources', marker_end=None), message='Starting initialization of resources [io_manager].', pid=15317, step_key='process_file_alias')
2021-02-25 16:14:17 - dagster - DEBUG - single_solid - df17c0a1-25d6-4a19-891a-5938d932ea60 - 15317 - process_file_alias - ENGINE_EVENT - Finished initialization of resources [io_manager].
DagsterEvent(event_type_value='ENGINE_EVENT', pipeline_name='single_solid', step_handle=StepHandle(solid_handle=SolidHandle(name='process_file_alias', parent=None)), solid_handle=None, step_kind_value=None, logging_tags={}, event_specific_data=EngineEventData(metadata_entries=[EventMetadataEntry(label='io_manager', description='Initialized in 0.01ms', entry_data=PythonArtifactMetadataEntryData(module='dagster.core.storage.mem_io_manager', name='InMemoryIOManager'))], error=None, marker_start=None, marker_end='resources'), message='Finished initialization of resources [io_manager].', pid=15317, step_key='process_file_alias')
2021-02-25 16:14:17 - dagster - DEBUG - single_solid - df17c0a1-25d6-4a19-891a-5938d932ea60 - 15317 - PIPELINE_START - Started execution of pipeline "single_solid".
DagsterEvent(event_type_value='PIPELINE_START', pipeline_name='single_solid', step_handle=None, solid_handle=None, step_kind_value=None, logging_tags={}, event_specific_data=None, message='Started execution of pipeline "single_solid".', pid=15317, step_key=None)
2021-02-25 16:14:17 - dagster - DEBUG - single_solid - df17c0a1-25d6-4a19-891a-5938d932ea60 - 15317 - ENGINE_EVENT - Executing steps in process (pid: 15317)
DagsterEvent(event_type_value='ENGINE_EVENT', pipeline_name='single_solid', step_handle=None, solid_handle=None, step_kind_value=None, logging_tags={}, event_specific_data=EngineEventData(metadata_entries=[EventMetadataEntry(label='pid', description=None, entry_data=TextMetadataEntryData(text='15317')), EventMetadataEntry(label='step_keys', description=None, entry_data=TextMetadataEntryData(text="['process_file_alias']"))], error=None, marker_start=None, marker_end=None), message='Executing steps in process (pid: 15317)', pid=15317, step_key=None)
tail: /Users/jai/Library/Application: No such file or directory
tail: Support/my-nextron-app/dagster_home/logs/df17c0a1-25d6-4a19-891a-5938d932ea60/compute_logs/process_file_alias.out: No such file or directory
tail: /Users/jai/Library/Application: No such file or directory
tail: Support/my-nextron-app/dagster_home/logs/df17c0a1-25d6-4a19-891a-5938d932ea60/compute_logs/process_file_alias.err: No such file or directory
2021-02-25 16:14:17 - dagster - DEBUG - single_solid - df17c0a1-25d6-4a19-891a-5938d932ea60 - 15317 - ENGINE_EVENT - Finished steps in process (pid: 15317) in 51ms
DagsterEvent(event_type_value='ENGINE_EVENT', pipeline_name='single_solid', step_handle=None, solid_handle=None, step_kind_value=None, logging_tags={}, event_specific_data=EngineEventData(metadata_entries=[EventMetadataEntry(label='pid', description=None, entry_data=TextMetadataEntryData(text='15317')), EventMetadataEntry(label='step_keys', description=None, entry_data=TextMetadataEntryData(text="['process_file_alias']"))], error=None, marker_start=None, marker_end=None), message='Finished steps in process (pid: 15317) in 51ms', pid=15317, step_key=None)
2021-02-25 16:14:17 - dagster - DEBUG - single_solid - df17c0a1-25d6-4a19-891a-5938d932ea60 - 15317 - PIPELINE_SUCCESS - Finished execution of pipeline "single_solid".
DagsterEvent(event_type_value='PIPELINE_SUCCESS', pipeline_name='single_solid', step_handle=None, solid_handle=None, step_kind_value=None, logging_tags={}, event_specific_data=None, message='Finished execution of pipeline "single_solid".', pid=15317, step_key=None)

```

---
#### Message from the maintainers:
Impacted by this bug? Give it a 👍. We factor engagement into prioritization.

**Comment by alangenfeld** [comment]
@prha

**Comment by prha** [comment]
This was resolved in `0.11.4`.
